core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/TaskInfo.java

```diff
@@ -50,17 +51,24 @@
 
     private final String description;
 
+    private final long startTime;
+
+    private final long runningTime;
```

Comment:
We either have to send it over the wire or synchronize clocks on all nodes. Otherwise, users might get negative running times, which just look weird. I think sending this over the wire is a simpler solution.